### PR TITLE
[ FilterBar ] Add display order for FilterBar Component

### PR DESCRIPTION
**Pull request by fti-pvishwakarma**
### Description

In current implementation Search Text field is always at first position, DatePickers are on second position and Dropdowns are on 3rd position if added in FilterBar. 
In this PR we have provided an enhancement in existing FilterBar to place Search text field, DatePickers and Dropdowns in user defined/specific order.

There was one more issues fixed (BH#27) when we change the language then in Search filter selected search text alignment was little bit up for Japanese.

Ref: 
https://www.bugherd.com/projects/281466/tasks/13
https://www.bugherd.com/projects/281466/tasks/27

image

 ### Reviewing/Testing steps

Follow the steps given at :
https://github.com/future-standard/scorer-ui-kit/

When project runs, Open http://localhost:3000/scorer-ui-kit/storybook/?path=/story/filters-organism--filter-bar in browser you will see
image

goto URL location scorer-ui-kit/storybook/?path=/story/filters-organism--filter-bar

change to sequence of array element to see effect

```
   displayOrder={['Dropdowns', 'Searchers', 'DatePickers']}
```

### CODE REVIEW

File: src/Filters/molecules/FilterInputs.tsx [hunk 1]
```diff
@@ -161,21 +172,28 @@ const FilterInputs: React.FC<IFilterInputs> = ({
...
       {/* {When the Dev does not initialize hasShowMore as true but has hidden inputs, it will show the add Searcher of the canHide} */}
       {(!hasShowMore || !showMoreDropdowns) && renderAddSearchButtons(searchFilters, visibleSearchInputs, handleVisibleSearch)}
       {hasShowMore && (
-        <FilterButton
-          icon='FilterEllipsis'
-          onClick={toggleShowMore}
-        >
-          {showMoreDropdowns ? showMoreText : showLessText}
-        </FilterButton>)}
+        <FilterButtonBox>
```
Comment (fti-bcarpenter) [inline src/Filters/molecules/FilterInputs.tsx:189]: Why this FilterButtonBox wrapper is added to FilterButton? I don't think this is needed? Any specific reason?

File: storybook/src/stories/Filters/organisms/FilterBar.stories.tsx [hunk 2]
```diff
@@ -250,10 +250,14 @@ export const _FilterBar = () => {
     setRows(rowMaker(data));
   }, [data])
 
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const dispalyOrder = object('Display Order', filterDisplayOrder);
```
Comment (fti-bcarpenter) [inline storybook/src/stories/Filters/organisms/FilterBar.stories.tsx:254]: Never disable the lint errors.

**Review by fti-bcarpenter — CHANGES_REQUESTED** [review]

File: storybook/src/stories/Filters/organisms/FilterBar.stories.tsx (hunk 2 above)
Comment (fti-pvishwakarma) [reply storybook/src/stories/Filters/organisms/FilterBar.stories.tsx]: Done

File: src/Filters/molecules/FilterInputs.tsx (hunk 1 above)
Comment (fti-pvishwakarma) [reply src/Filters/molecules/FilterInputs.tsx]: Done

**Review by fti-bcarpenter — APPROVED** [review]

**Comment by fti-pvishwakarma** [comment]
> 

I have successfully remove FilterbarPage in example folder

**Review by fti-bcarpenter — APPROVED** [review]
Looks good to me

**Comment by isacoder** [comment]
Thanks for the update, I will review the UI/UX part with @atomworks  and get back to you

**Comment by fti-bcarpenter** [comment]
> Thanks for the update, I will review the UI/UX part with @atomworks and get back to you

@isacoder Did you get a chance to review this PR with @atomworks ?

**Comment by isacoder** [comment]
Hello @fti-bcarpenter @fti-pvishwakarma 

Unfortunately after a lot of reflexion with the team about this approach of solving this issue we have decided not to proceed with it.

@atomworks will have a conversation with the person that requested in bugherd this change to have a better understanding of the problem before moving forward with a new solution.

Thank you for your patience and proactivity in following up this issue.

**Pull request closed without merging**